bin/logstash.lib.sh

```diff
@@ -1,4 +1,9 @@
-basedir=$(cd `dirname $0`/..; pwd)
+unset CDPATH
```

Comment:
When CDPATH env var is set `cd` becomes verbose about what directory was changed into. Here's an extract from the bash manual on CentOS 6:

> If a non-empty directory name from CDPATH is used, or if - is the first argument, and the directory change is successful, the absolute pathname of the new working directory is written to the standard output.

So unsetting CDPATH is good practice in shell scripts.